ros2nodl/ros2nodl/verb/validate.py

```diff
@@ -1,49 +1,49 @@
-# Copyright 2020 Canonical Ltd.
```

Comment:
This is still public API. Does it need to be?

Reply:
fixed in 70d6ba5